GetStream/stream-chat-android
# [CAS-273] Add support for channel slow mode

https://stream-io.atlassian.net/browse/CAS-273

### 🎯 Goal

Add support for channel slow mode

### 🎨 UI Changes

| Before | After |
| --- | --- |
| ![light](https://user-images.githubusercontent.com/9600921/130875114-be4800fe-7517-4242-8964-15f3a132aafb.png) | ![dark](https://user-images.githubusercontent.com/9600921/130875109-4dbb8ff1-4093-4cae-ac80-5b27abbc1cae.png) |

https://user-images.githubusercontent.com/9600921/130874904-cd950602-e983-438d-b59e-418e4a26a052.mov

### 🧪 Testing

1. Enable slow mode for a channel
2. Send a message
3. Observe cooldown countdown timer

### ☑️ Checklist


### 🎉 GIF

![giphy (7)](https://user-images.githubusercontent.com/9600921/130875354-8091a554-92f5-4bc1-8b13-0eccfe53b3b9.gif)
